web/src/components/Board.res

```diff
@@ -163,7 +163,7 @@ module Board = {
   type t = {title: string, columns: list<Column.t>, style: style}
 
   let defaultNegativeApprovals = "(approval:CHANGE_REQUESTED or approval:Verified-1 or approval:Workflow-1 or approval:Code-Review-1 or approval:Code-Review-2)"
-  let defaultPositiveApprovals = "(approval:APPROVED or approval:Code-Review+1 or approval:Code-Review+2)"
+  let defaultPositiveApprovals = "(approval:APPROVED or approval:Code-Review+2)"
```

Comment:
Why remove CR+1 ? This is used for the `To Review` list, not for the `To Approve` list

Reply:
I only want to see changes that received "real" approval in "to approve" and not in "to review" list.